kernel/kernel-api/src/main/java/io/delta/kernel/internal/tablefeatures/TableFeatures.java

```diff
@@ -652,7 +652,7 @@ public static boolean hasInvariants(StructType tableSchema) {
         .isEmpty();
   }
 
-  private static boolean hasCheckConstraints(Metadata metadata) {
```

Comment:
nit: lets move these one and the one above hasInvaraints to the public method section.